test/Conversion/TorchToMhlo/basic.mlir

```diff
@@ -295,3 +295,97 @@ func.func @torch.aten.cat(%arg0: !torch.vtensor<[?,?],f32>, %arg1: !torch.vtenso
   %1 = torch.aten.cat %0, %int0 : !torch.list<vtensor>, !torch.int -> !torch.vtensor<[?,?],f32>
   return %1 : !torch.vtensor<[?,?],f32>
 }
+
+// -----
+
+// CHECK-LABEL:   func.func @torch.aten.clamp(
```

Comment:
Recommend not including these tests:

https://github.com/llvm/torch-mlir/blob/main/docs/architecture.md#dos-and-donts-for-unit-vs-end-to-end-testing

> DON'T use a unit test if your lowering pattern could be described as a trivial "macro expansion" of one op into another op or set of ops. That is, if you feel like your unit test is just rewriting b.create<...>(...) into CHECK: ... then it is probably not a useful unit test.

Reply:
Okay, I understand the concerns. Sorry for bothering you with the same problem again. I will remove these mlir tests. Should I add e2e test for it?

Reply:
Yes, there must always be an e2e test for new ops.

Reply:
There might already be a e2e test if the ops are supported on one of the other backends.